cue/data/gen.cue

```diff
@@ -289,6 +290,102 @@ Family: scuemata.#Family & {
                         }]
                     }
                 }
+                // Row panel
+                #RowPanel: {
+                    type: "row"
+                    collapsed: bool | *false
```

Comment:
i also changed the panel definition to type: !="" & !="graph" & !="row", in this case, i make sure all definitions have no intersection.

Reply:
i'm confused about why this comment is anchored here

Reply:
i've updated over in the other spot, but only with `&& != "graph"` - i can't quite articulate why, but i'm not worried about `row` right now. maybe that's wrong :)

Reply:
had to revert that, it's not valid CUE. negations are hard.

unless/until we have a case that specifically shows us we need the additional exclusion of `graph` and `row` from `#Panel`, i don't think we should worry about it.